sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTest.cs

```diff
@@ -84,22 +91,74 @@ public void E2E_ThreadCreateUpdateGetDelete_MemberAddUpdateRemove_MessageGetSend
             var threadsCount = threads.Count();
 
             #region Snippet:Azure_Communication_Chat_Tests_E2E_SendMessage
-            SendChatMessageResult sendChatMessageResult = chatThreadClient.SendMessage("This is message 1 content", ChatMessagePriority.High, displayNameMessage);
+            string messageContent = "Content for message 1";
+            SendChatMessageResult sendChatMessageResult = chatThreadClient.SendMessage(messageContent, ChatMessagePriority.High, displayNameMessage);
```

Comment:
Is there a reason that we set the priority to `High`?

Reply:
nope. Any reason not to ? We can remove it , or set it to normal instead if it looks better.

Reply:
I'd stick to what is most likely the first thing being used by the SDK consumer. Probably not pass it or Normal I guess.